### PR TITLE
Add ncurses recipe that fixes ContinuumIO/anaconda-issues#455

### PR DESCRIPTION
**Pull request by MaxNoe**
This should fix ContinuumIO/anaconda-issues#455, at least for arch linux and every other system where the terminfo database is in `/usr/share/terminfo`.

If there are more places where `terminfo` database is often installed, I can also add them.

### CODE REVIEW

**Comment by msarahan** [comment]
Thanks @MaxNoe - let me clarify a bit about what conda-forge is.  This is a community effort, and Continuum is totally on board with it.  It is hopefully going to help the community make their improvements (like yours) more readily available without depending on Continuum to make them available in the defaults channel.  Continuum will be moving recipes out of the private space, into publicly-accessible spaces, and will pull from those public spaces when building packages for the defaults channel.  We haven't completely worked out how that happens yet.  Conda-forge operates on a one-repo-per-recipe model, and we (continuum) need new tools to work with that.  Additionally, recipes on conda-forge typically use jinja2 templating to reduce the maintenance burden.  Continuum's repository management currently does not take templating into account.

TLDR: this is not simply a replacement for conda-recipes.  It is quite a bit more.  Thanks for your contribution here, and I hope you'll help us in this effort with your feedback.

**Comment by jakirkham** [comment]
Thanks @MaxNoe. So, we actually have ncurses already, but would be happy to look at any fixes that you propose. Could you please apply your patches in a PR against the [ncurses feedstock]( https://github.com/conda-forge/ncurses-feedstock )?

**Comment by MaxNoe** [comment]
Quite and odessey, i hope everything is now in the correct place.

**Comment by msarahan** [comment]
Indeed, my apologies for not searching the feedstocks first and pointing you in the right place. =(

Thanks again for your patience and effort.

**Comment by pelson** [comment]
Thanks for taking the time to do this @MaxNoe - I'm sorry that it turned out there was already an ncurses feedstock, but glad you've put forwards https://github.com/conda-forge/ncurses-feedstock/pull/1! 👍 

Out of interest, did you take a look at https://conda-forge.github.io/feedstocks.html before packaging ncurses? If not, would it have been helpful if that list were more prominent? We have an issuehttps://github.com/conda-forge/conda-forge.github.io/issues/74 which would really benefit from your recent experience.

Thanks @MaxNoe!

**Comment by MaxNoe** [comment]
@pelson Althugh I'm a heavy conda user and look into the blog quite often I never heard of conda-forge before ;)

**Comment by pelson** [comment]
> Great Project! How is this connected to ht actual conda builds? Are these still closed source?

Do you mean the ones provided by the "defaults" channel from Continuum? As @msarahan says "Continuum will be moving recipes out of the private space, into publicly-accessible spaces" but currently that is not the case (I don't work there, and don't know their timescales, but @msarahan is making extremely valuable contributions to the conda-forge [open] community, and that is being supported by Continuum).

Conda-forge is a community led solution to the open and reproducible build problem. Everything on conda-forge is open source (recipes, build infrastructure, feedstock orchestration etc.). Our aspiration is for conda-forge to be the canonical place that people come to use, and contribute to, the full application stack of conda distributions.

**Comment by jakirkham** [comment]
Everything is in the right place now, @MaxNoe. Thanks for going to the effort of figuring out this bug and coming up with a fix. Let's see what we can do to integrate this and get a working release out to you.

**Comment by msarahan** [comment]
Also note that all internal recipes which have been converted to conda recipes are available at https://github.com/ContinuumIO/anaconda-recipes - this is about 250 of 650 total internal recipes.  it is still my goal to move all recipes to a single canonical source that is under community control - IMHO, either at conda-forge, or at the original package's source (ideally, with conda-forge or some other automation using this source, or a submodule of it to build packages).
